src/PowerShellEditorServices/Services/TextDocument/Handlers/CodeLensHandlers.cs

```diff
@@ -33,20 +33,22 @@ public class CodeLensHandlers : ICodeLensHandler, ICodeLensResolveHandler
         private readonly ILogger _logger;
         private readonly SymbolsService _symbolsService;
         private readonly WorkspaceService _workspaceService;
+        private readonly ConfigurationService _configurationService;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_configurationService' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4894668)